api/src/main/java/org/openmrs/notification/SentMessage.java

```diff
@@ -0,0 +1,202 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.notification;
+
+import java.util.Date;
+
+import org.openmrs.User;
+
+/**
+ * This is used by the System administrator to view all sent messages stored in the 
+ * database. For normal messaging functions use the Message api
+ * 
+ * @see org.openmrs.notification.Message
+ * @see org.openmrs.notification.MessageService
+ */
+public class SentMessage {
+	
+	private Integer messageId;
+	
+	private Integer senderId;
```

Comment:
I was trying to follow the requirements on the ticket, because it looked liked the requester wanted to persist only the ids of the sender and the receiver.